runtime/doc/starting.txt

```diff
@@ -411,12 +411,7 @@ accordingly, proceeding as follows:
 
 5. Enable filetype and indent plugins.
 	This does the same as the command: >
-		:filetype plugin indent on
-<	which in turn is equivalent to: >
-		:runtime! filetype.lua
-		:runtime! filetype.vim
-		:runtime! ftplugin.vim
-		:runtime! indent.vim
+		:runtime! ftplugin.vim indent.vim
```

Comment:
no Lua?

Reply:
No, we don't support `ftplugin.lua` or `indent.lua` yet.